plugins/PostProcessingPlugin/scripts/Stretch.py

```diff
@@ -83,6 +86,24 @@ def copyPosFrom(self, step):
     def setInRelativeMovement(self, value: bool) -> None:
         self.in_relative_movement = value
 
+class GCodeStepSlot:
```

Comment:
Perhaps we can just use a `namedtuple` for this instead?

Reply:
Good idea!